pkg/middlewares/accesslog/logger.go

```diff
@@ -193,6 +193,11 @@ func (h *Handler) ServeHTTP(rw http.ResponseWriter, req *http.Request, next http
 	core[RequestPath] = urlCopyString
 	core[RequestProtocol] = req.Proto
 
+	core[RequestScheme] = "http"
+	if req.TLS != nil {
+		core[RequestScheme] = "https"
+	}
```

Comment:
what about ws and wss? do we want to take them into account? do we want to look at the headers to see the connection upgrade maybe?

Reply:
nevermind, it's probably irrelevant.